Yoda/yoda.h

```diff
@@ -126,18 +128,21 @@ struct APIWrapper : APICalls<YodaTypes<ENTRIES_TYPELIST>> {
 };
 
 // `yoda::API` suports both a typelist and an `std::tuple<>` with parameter definition.
-template <typename... SUPPORTED_TYPES>
+template <template <typename> class PERSISTENCE, typename... SUPPORTED_TYPES>
 struct APIWrapperSelector {
-  typedef APIWrapper<std::tuple<SUPPORTED_TYPES...>> type;
+  typedef APIWrapper<PERSISTENCE, std::tuple<SUPPORTED_TYPES...>> type;
 };
 
-template <typename... SUPPORTED_TYPES>
-struct APIWrapperSelector<std::tuple<SUPPORTED_TYPES...>> {
-  typedef APIWrapper<std::tuple<SUPPORTED_TYPES...>> type;
+template <typename... SUPPORTED_TYPES, template <typename> class PERSISTENCE>
```

Comment:
Yep, my question was about consistency - the only place where PERSISTENCE goes last is line 136, that's why I was asking.